lib/diagnostic.js

```diff
@@ -14,9 +23,12 @@ const min = function (/* previous, current */) {
 // the sum of the values in the array
 // this callback will be passed by the tests to the reduce method, e.g.
 // `let total = array.reduce(sum);`
-const sum = function (/* previous, current */) {
+const sum = function (previous, current) {
+  let array = new Array(function( ){
+    for (let i = 0;)
+  };
 
-};
+});
 
 // All of the callbacks in questions 3 through 5 should
 // be called with the current element in the array,
```

Comment:
Here are the solutions for one and two. Please review the solutions and run them to see what's happening. We'll be going over some of this after lunch as well 👍  

```js
// Question 1
// Write a callback for reduce that selects
// the minimum value in an array
// this callback will be passed by the tests to the reduce method, e.g.
// `let minimum = array.reduce(min);`
const min = function (previous, current) {
  return previous < current ? previous : current;
};

// Question 2
// Write a callback for reduce that produces
// the sum of the values in the array
// this callback will be passed by the tests to the reduce method, e.g.
// `let total = array.reduce(sum);`
const sum = function (previous, current) {
  return previous + current;
};
```